attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/AttributeServiceImpl.java

```diff
@@ -328,7 +332,11 @@ private void sendResult(
 
   private Query getQueryForFilter(
       String tenantId, AttributeMetadataFilter attributeMetadataFilter) {
-    List<AttributeScope> scopeFilterRequest = attributeMetadataFilter.getScopeList();
+    List<String> scopeFilterRequest =
```

Comment:
nit: `s/scopeFilterRequest/scopeFilterList/`?

Reply:
I noticed that, commented to myself that the existing naming was bad, and left it as is. Let me fix.